src/js/endpoint.js

```diff
@@ -6,22 +6,27 @@ module.exports.rootUrl = ROOT_URL;
 
 // State params set by main
 module.exports.appToken = undefined;
+module.exports.jwt = undefined;
 module.exports.appUserId = undefined;
 
 module.exports._rest = function(method, path, body) {
     var deferred = $.Deferred();
+    var headers = {
+        'app-token': this.appToken
+    };
+
+    this.jwt ? headers.Authorization = 'Bearer ' + this.jwt : '';
```

Comment:
Right... But I don't want the alternative you mentioned because that still adds the key to the Object.